### ISSUE TITLE
Bad link to MathML `altimg` guidance

### ISSUE THREAD

**Issue by dauwhe** [w3c/epub-specs#1268]
Changes doc has

```html
<p>Authors are instead directed to the existing guidance on the use of the <a href="epub-contentdocs.html#sec-xhtml-mathml-alt">MathML <code>altimg</code> attribute</a>.</p>
```
But there's no discussion of `altimg` in content docs. Should we just link to MathML itself?

**Comment by dauwhe** [comment]
3.1 had

> As Reading System support for MathML rendering is inconsistent, Authors are encouraged to provide a fallback image using the altimg attribute on the math element. It is recommended that the dimension and alignment attributes (altimg-width, altimg-height and altimg-valign) be used in conjunction with the altimg attribute.

I thought we got rid of this because talking about fallbacks for core media types was weird?

**Comment by mattgarrish** [comment]
Yes, the advice is out of whack now. We were just offering how to include an image of the math in the absence of epub:switch, which was an internal reference because we already had prose we could refer to.

I'm not sure there's value in this case in making any recommendation, as there are various methods you could use to provide fallbacks for math, and altimg probably isn't near the top of them.

**Comment by dauwhe** [comment]
I'm just removing the sentence.

**Comment by dauwhe** [comment]
Fixed via #1269
